.cicd/scripts/srw_unittest.sh

```diff
@@ -0,0 +1,32 @@
+#!/user/bin/env bash
+#
+# A test script for running the SRW application unittest tests that
+# should be tested on-prem.
+#
+set -e -u -x
+
+script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
+
+# Get repository root from Jenkins WORKSPACE variable if set, otherwise,
+# set relative to script directory.
+declare workspace
+if [[ -n "${WORKSPACE}" ]]; then
+  workspace="${WORKSPACE}"
+else
+  workspace="$(cd -- "${script_dir}/../.." && pwd)"
+fi
+
+# Only run this on machines with hpss access
+hpss_machines=( jet hera )
+if [ $hpss_machines =~ ${SRW_PLATFORM} ] ; then
+
+  module load hpss
+  module use ${workspace}/modulefiles
+  module load wflow_${SRW_PLATFORM}
+
+  conda activate regional_workflow
```

Comment:
On several machines, issues are encountered while attempting to activate the regional_workflow conda environment through the script.  To get around this, please use:

```suggestion
  source ${workspace}/ush/load_modules_wflow.sh ${SRW_PLATFORM}
  module load hpss
```

Reply:
@MichaelLueken I am trying on hera to get this to work. You finally opened my eyes that I can test this file locally (sorry I was putting you through all this!)

I was not able to get the conda environment loaded sufficiently on hera with this new source command, either.

Do you have any other tricks?

Reply:
@christinaholtNOAA No worries!  Another trick to load the conda environment is:

```
  module load hpss
  module use ${workspace}/modulefiles
  module load wflow_${SRW_PLATFORM}

  set +e +u
  conda activate regional_workflow
  set -e -u
```

I was able to get this method to load the conda environment on Hera.

Reply:
Never mind! I found the issue. It did not play well with me running this script in a shell that already had a conda environment loaded...even though that environment also has the needed packages. Weird.